SECRETS.md

```diff
@@ -47,7 +49,6 @@ curl -X POST -d \
 
 Now update the function to use the new secret:
 
-
 ```
 curl -X POST -d \
```

Comment:
I feel like we should remove the `curl` example now, it was there because the CLI didn't support secrets in the YAML.